src/module/sheet-skill-actions.ts

```diff
@@ -42,52 +43,61 @@ Hooks.on('renderActorSheet', async (app: ActorSheet, html: JQuery<HTMLElement>)
 
   const tpl = 'modules/pf2e-sheet-skill-actions/templates/skill-actions.html';
 
-  const skillActions: Array<SkillAction> = initializeSkillActions(app.actor).sort((a, b) => {
-    return a.label > b.label ? 1 : -1;
-  });
-  const skillActionHtml = $(await renderTemplate(tpl, { skills: skillActions }));
-
-  skillActionHtml.on('click', '.skill-action.tag.variant-strike', function (e) {
-    if (!(game instanceof Game)) return;
-
-    const modifiers = [];
-    const map = parseInt(this.dataset.map);
-
-    if (map) {
-      modifiers.push(
-        new ModifierPF2e({
-          label: game.i18n.localize('PF2E.MultipleAttackPenalty'),
-          modifier: map,
-          type: 'untyped',
-        }),
-      );
-    }
-    game.pf2e.actions[this.id]({ event: e, modifiers });
+  const skillActions = initializeSkillActions(app.actor);
+
+  const allVisible = Flag.get(app.actor, 'allVisible') ?? true;
```

Comment:
Not sure if it's better:
1. Make all actions visible and hide hidden actions by default
2. Or make all actions hidden but show hidden actions by default

It basically comes down to whether it's easier for user to unequip unused action or equip used actions . I implemented 2, but not that I'm writing this I'm starting to lean towards 1.

Reply:
I think I like 1 also. I think it's a little easier to clean up what you don't want rather than add things later

Reply:
Ok, I'll update to 1

Reply:
Updated